tools/buildtools/ambertools/parmconv.xml

```diff
@@ -198,6 +248,10 @@ for system in systems:
       <param name="stripmask_receptor" value=":NA,CL,SOL,WAT"/>
       <!-- test a fairly complex selection. All backbone oxygens in residues 1-500 but not in water, lysine or arginine -->
       <param name="stripmask_solvatedcomplex" value=":1-500@O&amp;!(:WAT|:LYS,ARG)"/>
+      <assert_stdout>
+        <has_text text="Getting coordinates"/>
+        <has_text text="Using topology directory"/>
+      </assert_stdout>
       <output name="prmtop_ligand">
```

Comment:
Should we add a new test for the new remove dihedral/periodicity options?

Reply:
Good idea to double check that it work :)